packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardTypes.ts

```diff
@@ -0,0 +1,50 @@
+export type ReleaseCardStat = {
+  label: string
+  value: string
+}
+
+export type StatConfig = {
+  // specifies which column should be used to populate the statistic value
+  columnName: string
+  // the label that will be displayed with the statistic in the card
+  label: string
+}
+
+export type ButtonToExplorePageConfig = {
+  // button's label
+  label: string
+  // column name in the data (table, view...) driving the page where the button is located
+  // which is used to populate the button's path
+  sourcePathColumnName: string
+  // defining SQL for the data (table, view...) driving the Explore page
+  // which is used in generating the button's query string
+  exploreDataSql?: string
+  // column name of the selected facet in the data (table, view...) driving the Explore page
+  // which is used in generating the button's query string
+  exploreDataFacetColumnName?: string
+  // column name in the data (table, view...) driving the page where the button is located
+  // which is used to to populate the value for the selected facet in the button's query string
+  sourceDataFacetValueColumnName?: string
+}
+
+export type ReleaseCardLargeConfig = {
+  cardSize: 'large'
+  // whether the release version should be prefixed with "Release "
+  prependRelease: boolean
+  // statistics to display in the release card
+  statsConfig: StatConfig[]
+  // button to explore data in the release
+  buttonToExplorePageConfig?: ButtonToExplorePageConfig
+  // path to release's data guide
+  dataGuidePath?: string
+}
+
+export type ReleaseCardMediumConfig = {
+  cardSize: 'medium'
+  // path to request access to the release data
+  requestAccessPath: string
```

Comment:
In the [Figma](https://www.figma.com/file/BI4y33EHA95onN8DourTNZ/Two-Projects?type=design&node-id=263-15875&mode=design&t=76oHvfvp9FWFtDSR-4), previous release cards all point to the same "Request Access" page. If we don't expect that medium release cards will always point to a "Request Access" page, I could change this field name to be more generic (e.g. "cardLinkPath"?).